app/services/customers/update_service.rb

```diff
@@ -6,6 +6,10 @@ def update(**args)
       customer = result.user.customers.find_by(id: args[:id])
       return result.not_found_failure!(resource: 'customer') unless customer
 
+      unless valid_metadata_count?(metadata: args[:metadata])
```

Comment:
maybe we can validates this with a validator on the model? wdyt?

Reply:
I am still not sure what is the final decision about validator classes, is it a rule of thumb to use it across the project? I know that some of us don't prefer it but some like it 😄 I personally haven't used it here since it is just one condition that needs to be checked on service level but if having validators are style that we are going to follow, I can change it so that we are consistent:)

Reply:
@jdenquin I talked about model validation with Vincent and Romain before and we concluded that such a validation is more business related and that we should place it service level as other similar validations. Do you agree with that?